lighthouse-core/config/constants.js

```diff
@@ -103,6 +103,8 @@ const defaultSettings = {
   screenEmulation: screenEmulationMetrics.mobile,
   emulatedUserAgent: userAgents.mobile,
 
+  displayStrings: {},
```

Comment:
not the prettiest thing to default to `{}`, but after discussion we think maybe this is the best compromise given the constraints of the `Settings` type today. It would be kind of nice to default to not defined if there are no strings in it, but semantically I think it's still ok (the empty set of display strings)